ansible/plan.sh

```diff
@@ -0,0 +1,51 @@
+pkg_name=ansible
+pkg_origin=core
+pkg_version="2.4.3.0-1"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("GPL-3.0")
+pkg_source="https://github.com/${pkg_name}/${pkg_name}/archive/v${pkg_version}.tar.gz"
+pkg_filename="${pkg_name}-${pkg_version}.tar.gz"
+pkg_shasum="0bcb583f2abaa580c337462a81903ff3f925d10c367da466c2c0354479432683"
+pkg_deps=(
+  core/libffi
+  core/python2
+  jirotech/sshpass
```

Comment:
I'm not sure how we feel about pulling from third party packages in `core`, but this `sshpass` plan is about as bare as it can be. I wonder how jirotech feels about contributing it into `core`.

Reply:
Taking a look at our current standards, we have nothing about not allowing third party packages in core https://github.com/habitat-sh/core-plans/blob/master/CONTRIBUTING.md.  I'm personally ok with it.

Reply:
I'm pretty concerned about this actually. I think that a core plan implies a degree of testing and validation that is not true of third party origins, and so the dependencies of a core plan ought to be subject to the same degree of testing - ie, that they're also a core plan. 

I can also see in the future on premise hab installations that choose to only mirror core, which would cause issues.

Reply:
I also am 👎 on allowing external dependencies into core packages.

Reply:
I'm glad this is spurring discussion. So, I could submit a core plan for sshpass, but it would probably look identical to the third-party plan. Are there licensing concerns with that? Maybe we ought to reach out to the OP and see if we can get it contributed to core.